Cargo.toml

```diff
@@ -20,3 +20,13 @@ eframe = { version = "0.28.0", default-features = false, features = [
     "glow",
 ] }
 egui-phosphor = { git = "https://github.com/ItsEthra/egui-phosphor", branch = "main" }
+
+[lints.rust]
+unsafe_code = "forbid"
+
+[lints.clippy]
+nursery = { level = "deny", priority = 0 }
+pedantic = { level = "deny", priority = 1 }

```

Comment:
Really don't like pedantic and nursery group, a bunch of annoying and opinionated lints. Remove it.

Afterwards `module_name_repetitions`, `cast_precision_loss` can be removed as well because they are allow by default AFAIK.